features/nested_spans.feature

```diff
@@ -20,7 +21,7 @@ Feature: Nested Spans
     * the trace payload field "resourceSpans.0.scopeSpans.0.spans.0.traceId" is stored as the value "parent_trace_id"
     * the trace payload field "resourceSpans.0.scopeSpans.0.spans.1.traceId" equals the stored value "parent_trace_id"
 
-
+  @skip_webgl
```

Comment:
I take it from this that we're not supporting contexts/nested spans with WebGL? We should make sure to document that in online docs and also – I'd suggest – the PR description and perhaps a comment alongside the `@skip` to help us remember why.